omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,418 @@
+restricted privileges using the OMERO.web
+interface, see :help:`create new users <sharing-data#admin>` chapter
+of our Help documentation.
+
+We suggest here four setups that should cover the four mainstream
+workflows. Nevertheless, you can combine the privileges
+(check the checkboxes in the OMERO.web interface)
+in any way you see fit. The privilege were designed in such a way
+that they still bear useful functionality even when used in
+isolation. For example checking the :term:`Chown` checkbox will
+indeed give the new administrator with restricted
+privileges the power to transfer ownership of other users'
+data with no other ifs and buts.
+
+Four suggested workflows
+------------------------
+
+=============================== ======================= ======================= ===================== ================================
+Required Privileges             :term:`Data viewer`     :term:`Importer`        :term:`Analyst`       :term:`Group and Data Organizer`
+------------------------------- ----------------------- ----------------------- --------------------- --------------------------------
+:term:`Sudo`                                      N                     Y                      N                       N
+:term:`Write Data`                                N                     N                      Y                       Y
+:term:`Delete Data`                               N                     N                      N                       Y
+:term:`Chgrp`                                     N                     N                      N                       Y
+:term:`Chown`                                     N                     N                      Y (O)                   Y
+:term:`Create and Edit Groups`                    N                     N                      N                       Y
+:term:`Create and Edit Users`                     N                     N                      N                       Y
+:term:`Add Users to Groups`                       N                     N                      N                       Y
+:term:`Upload Scripts`                            N                     N                      Y                       N
+
+=============================== ======================= ======================= ===================== ================================
+
+Y == privilege required, checkbox in OMERO.web interface is checked
+N == privilege not required, checkbox is not checked
+(O) == this privilege is suggested as optional for the workflow
+
+.. note::
+    **Restricted admins workflows in OMERO.clients:** Please do not
+    expect for any workflows mentioned here that all OMERO.clients
+    OMERO.web, OMERO.insight, command line interface (CLI)) are fully
+    equipped to execute them (see details below). New features will
+    be added in OMERO.clients in the 5.3.x series of OMERO releases.
+
+.. note::
+    **Group allegiances** All the workflows here do not assume that
```

Comment:
`do not assume` and `is not a member`: the double negative is confusing; could this be stated as a positive?